pydeseq2/utils.py

```diff
@@ -73,16 +73,16 @@ def load_example_data(
     if dataset == "synthetic":
         path_to_data = datasets_path / "synthetic"
         if Path(path_to_data).is_dir():
-            path_to_data_counts = path_to_data / "test_counts.csv"
-            path_to_data_clinical = path_to_data / "test_clinical.csv"
+            path_to_data_counts = str(path_to_data / "test_counts.csv")
```

Comment:
I tested if it passes for the is_dir True when you remove changing to string and it passes (path_to_data_counts is <class 'pathlib.PosixPath'> and that's ok for numpy. ie no need to set to str in this case. The numpy fails if url is a Path (which makes sense) but if it's a path within the directories it's fine

Reply:
Indeed, but `mypy` is not happy when `path_to_data_counts` can either be a `Path` or a `str`, so I cast it to `str` which is the common denominator

Reply:
ok, in this case maybe just add a comment on why it is done like this

Reply:
Done